Samurai/Modules/ModuleManager.cs

```diff
@@ -1,17 +1,16 @@
-using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
 using System.Threading.Tasks;
+using AcidChicken.Samurai.Assets;
+using AcidChicken.Samurai.Components;
```

Comment:
All of the local namespace reference instructions have to be included in the local namespace block.